cmd/atlas/internal/cmdapi/cmdapi.go

```diff
@@ -23,6 +23,32 @@ var (
 		Use:          "atlas",
 		Short:        "A database toolkit.",
 		SilenceUsage: true,
+		PostRunE: func(cmd *cobra.Command, _ []string) error {
```

Comment:
PostRun will block the CLI. Start the check in the background and on PostRun check if there's something to report. 

I'd say that it should be enabled only in development (not all operations, e.g. schema/migrate apply).